rust/linux-client/src/main.rs

```diff
@@ -144,7 +144,7 @@ struct Cli {
     #[command(flatten)]
     common: CommonArgs,
 
-    /// Identifier generated by the portal to identify and display the device.
+    /// Identifier used by the portal to identify and display the device.
```

Comment:
@jamilbk I noticed in `firezone-linux-client --help` that this description was backwards so I put that in the PR too